tests/networks/test_build.py

```diff
@@ -56,3 +58,59 @@ def test_directed_network():
     assert (
         directed_network.number_of_edges() == 2 * undirected_network.number_of_edges()
     )
+
+
+def test_jaccard_similarity():
+    """tests that when the jaccard similarity is used as an edge attribute it returns the correct value"""
+    sequence = [
+        ["I", "went", "to", "the", "party"],
+        ["i", "had", "fun", "at", "the", "party"],
+    ]
+    network = build_coocc(sequence, edge_attributes=["jaccard_similarity"])
+    attrs = nx.get_edge_attributes(network, "jaccard_similarity")
```

Comment:
Should there be a test to make sure that the attribute gets created separately from it being the correct value?